demo/src/main/java/com/restaurant/demo/exception/ApiExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.restaurant.demo.exception;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
+
+	@ExceptionHandler(value = {ApiRequestException.class})
+	public ResponseEntity<Object> handleApiRequestException(ApiRequestException e){
+		HttpStatus badrequest = HttpStatus.BAD_REQUEST;
```

Comment:
rename badrequest to badRequest, don't forget to use camel case format

Reply:
https://www.ionos.com/digitalguide/hosting/technical-matters/the-most-important-http-status-codes-at-a-glance/